assets/js/shared/context/inner-block-layout-context.tsx

```diff
@@ -20,11 +19,17 @@ const InnerBlockLayoutContext = createContext( {
 export const useInnerBlockLayoutContext = () =>
 	useContext( InnerBlockLayoutContext );
 
+interface InnerBlockLayoutContextProviderProps {
+	parentName?: string;
+	parentClassName?: string;
+	children: React.ReactNode;
+}
+
 export const InnerBlockLayoutContextProvider = ( {
 	parentName = '',
 	parentClassName = '',
 	children,
-} ) => {
+}: InnerBlockLayoutContextProviderProps ) => {
 	const contextValue = {
 		parentName,
 		parentClassName,
```

Comment:
I noticed that TS now shows an error about this object. It seems that missing the `isLoading` value. We can fix this issue in two ways:
- put isLoading in this object.
- redefine the context in this way:

```const InnerBlockLayoutContext = createContext< {
	parentName: string;
	parentClassName: string;
	isLoading?: boolean;
} >( {
	parentName: '',
	parentClassName: '',
	isLoading: false,
} );
```